src/script/components/list/GroupedConversations.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+
+import type {ConversationRepository} from '../../conversation/ConversationRepository';
+import type {Conversation} from '../../entity/Conversation';
+import {
+  ConversationLabel,
+  createLabel,
+  createLabelFavorites,
+  createLabelGroups,
+  createLabelPeople,
+} from '../../conversation/ConversationLabelRepository';
+import {generateConversationUrl} from '../../router/routeGenerator';
+import {createNavigate} from '../../router/routerBindings';
+
+import './GroupedConversationHeader';
```

Comment:
This can go, right?

Reply:
https://github.com/wireapp/wire-webapp/pull/11621/commits/42b92f2df802505f110ebebad38a3dc30b46c3f0